setup.py

```diff
@@ -30,8 +30,6 @@
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
         'Framework :: Django',
-        'Framework :: Django :: 1.8',
-        'Framework :: Django :: 1.9',
```

Comment:
isn't Django 1.8+ supported by these changes?

Reply:
Yes indeed, forgot to remove that, removed now :)